### DISCUSSION TITLE
New user problems

### DISCUSSION THREAD

**Discussion by nelson777** [zed-industries/zed#15586] [Feature Requests]
Hello all,

Recently I heard about Zed and decide to take it for a walk. So in this walk I got a very good impression on Zed and liked it a lot. 
I love the AI tools readily available, vim mode, git blame, small icons that get off your way and simply the way it handles things. Also collaboration is AWSOME. So I had a very good first impression.

But... this isn't romance, this is real life and for a professional developer like me used to the conforts of Eclipse, it's not so simple to adopt another IDE just like that. It needs to support at least the basics. So I got the following issues. Could you please enlighten me if these are my mistakes or if there is something really missing:

1- I had problems with accents in portuguese like ç apears like ć in te editor. This is not a problem with the system (Linux Mint Vanessa), because it works fine everywhere else. This happens anywhere in Zed: editor, input fields, chat, everywhere.

2- Git integration. I saw branch selection, git blame and... pretty much it. If it's just this, then it's a show stopper. I need (at least): changed and staged files, git diff, git history, push and pull from repo, git ignore, that I can remember from my head right now. I can't affort not to have any of these.
In Zed's site it's stated that it integrates with several online git services like github, gitlab and others. How ? I couldn't find any configuration for git online services.

3- File comparision: how do I compare 2 files ? 

4- I use VIM heavily. How compatible is VIM mode ? g command, regex, macros... what isn't available ?

Thanks. I wish a bright future for Zed. :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
